common/roxiecommlib/roxiecommunicationclient.cpp

```diff
@@ -321,7 +321,7 @@ class CRoxieCommunicationClient: implements IRoxieCommunicationClient, public CI
     {
         unsigned start = msTick();
         if (!sendRoxieControlLock(sock, allOrNothing, wait))
-            throw MakeStringException(-1, "Roxie control:lock failed");
+            throw MakeStringException(-1, "Roxie is too busy - please try again later.");
```

Comment:
I think worth preserving the fact that it was a control lock failure (it helps me tell people what went wrong) while adding the information that it indicates roxie is busy and should try again later.

e.g.
"Roxie is too busy (control:lock failed) - please try again later"